### ISSUE TITLE
Update to at least Node 16

### ISSUE THREAD

**Issue by calexander3** [bump-sh/github-action#322]
Using the action now results in the following warning:
```
Node.js 12 actions are deprecated. For more information see: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/. Please update the following actions to use Node.js 16: bump-sh/github-action
```

**Comment by paulRbr** [comment]
Thanks @calexander3 for the feedback :slightly_smiling_face: we have released [v1.1.5](https://itducks.slack.com/archives/C7TBKDEC8/p1665581505781409) with the upgrade.

**Comment by calexander3** [comment]
Thanks @paulRbr! I appreciate the quick turn around!
